### DISCUSSION TITLE
Interpreting Gene/Peak Link Plots

### DISCUSSION THREAD

**Discussion by AAA-3** [stuart-lab/signac#925] [Q&A] [answered]
Hello everyone-

Just had a quick question to  check how to interpret the following plot correctly (from vignette):
![image](https://user-images.githubusercontent.com/68068335/147267466-c2a92ca2-f3e0-41b3-b7d3-1f5004ae4f7c.png)

Is my understanding correct?:

- The top half shows normalised accessibility of each peak corresponding to the gene for each cluster, with the range coming from the normalisation calculations of each gene
- Underneath the accessibility is the gene coding region mapped to the relevant chromatin position
- This is followed by the Peaks shown in grey which were retrieved from MACS2

I am not sure how to interpret the links: I understand the colour corresponds to how strongly the peak correlates with gene expression, but I do not know why peaks detected after the coding region of the gene do not show any linking. And the what is the score in the side legend represent? Z-score of pearson correlation? And how do I interpret the violin plots on the top right (What is the x-axis?)?

Apologies if this has already been explained else where or in the paper...

Happy holidays!

**Comment by timoast** [comment, marked as answer]
> The top half shows normalised accessibility of each peak corresponding to the gene for each cluster, with the range coming from the normalisation calculations of each gene

Not quite, the accessibility tracks show the normalized fragment counts per group of cells per basepair. The normalization is for the different number of cells in each group and different average sequencing depth per group of cells. It doesn't have anything to do with the gene annotations shown.

> Underneath the accessibility is the gene coding region mapped to the relevant chromatin position

Correct

> This is followed by the Peaks shown in grey which were retrieved from MACS2

Correct (if you called peaks using MACS2)

> I do not know why peaks detected after the coding region of the gene do not show any linking

There is a p-value and score cutoff for detecting a peak-gene link in the `LinkPeaks()` function (0.05; https://satijalab.org/signac/reference/linkpeaks)

> And the what is the score in the side legend represent

Correlation coefficient between peak and gene (see documentation for LinkPeaks)

> And how do I interpret the violin plots on the top right (What is the x-axis?)?

This is the normalized expression of the features plotted. The units will depend on the assay and how it was normalized
